### PR TITLE
Fix url search in distributions

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
`WbApplicationInfo::repo()` and `WbApplicationInfo::branch()` rely on text files that are not included in the distribution, hence if searching for the file locally fails, the search of its remote counterpart will fail as well since the url is malformed.

### CODE REVIEW

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

File: src/webots/nodes/utils/WbUrl.cpp [hunk 1]
```diff
@@ -122,8 +122,10 @@ QString WbUrl::computePath(const WbNode *node, const QString &field, const QStri
 
     if (isLocalUrl(url)) {
       QString newUrl(url);
-      newUrl.replace("webots://", "https://raw.githubusercontent.com/" + WbApplicationInfo::repo() + "/" +
-                                    WbApplicationInfo::branch() + "/");
+      WbVersion version = WbApplicationInfo::version();
```
Comment (BenjaminDeleze) [inline src/webots/nodes/utils/WbUrl.cpp:125]:
```suggestion
      WbVersion version = WbApplicationInfo::version();
```

Would it be possible to make a const variable here ?

Comment (ad-daniel) [reply src/webots/nodes/utils/WbUrl.cpp:125]: Yes, both can actually

**Review by BenjaminDeleze — APPROVED** [review]
Thank you

**Pull request merged**